cmd/lava/internal/help/helpdoc.go

```diff
@@ -282,3 +284,89 @@ General-purpose environment variables:
 		considered experimental.
 	`,
 }
+
+// HelpChecktypes documents the Lava checktypes file.
+var HelpChecktypes = &base.Command{
+	UsageLine: "checktypes",
+	Short:     "checktype catalog file",
+	Long: `
+Checktypes are programs that integrate third-party tools (e.g. Trivy,
+Semgrep, etc.) or implement a custom detection for a given
+vulnerability. On the other hand, a check is a concrete instance of a
+checktype that is run against a specific target with a specific set of
+options.
+Enabled checktype catalogs are configured in the Lava configuration
+file. For more details, use "lava help lava.yaml".
```

Comment:
We forgot to introduce what is a checktype catalog.

```suggestion
A checktype is a program that integrates a third-party tool
(e.g. Trivy, Semgrep, etc.) or implements a custom detection for a
given vulnerability. A check is a concrete instance of a checktype
that is run against a specific target with a specific set of
options.

Checktypes are organized in catalogs. A checktype catalog is a
collection of checktypes with their metadata and default options. This
help topic describes the checktype catalog format in detail.

The list of enabled checktype catalogs is specified in the Lava
configuration file. For more details, use "lava help lava.yaml".
```